### ISSUE TITLE
Feature request: data aggregation

### ISSUE THREAD

**Issue by JohnnyDoorn** [jasp-stats/jasp-desktop#1956]
## Template for enhancement requests
 
* Enhancement: allow for data aggregation
* Purpose:
I just received a request at ESCoP, and it wasn't the first time this was brought up:
Some researchers would like to be able to use JASP for some preprocessing of their data. For instance, these researchers have 500 reaction times per participant per condition, and they would like to take the average per condition per participant, to enable further analysis (e.g., anova or t-test). Are we already working on a similar feature?
Of course this could be done in Excel or R (or even SPSS), but it kind of defeats the purpose of getting people fully aboard the JASP train ;)

**Comment by AlexanderLyNL** [comment]
Hi Johnny, can you provide a use case? That'll help the process.

**Comment by EJWagenmakers** [comment]
I think this also relates to the request of being able to switch from long to wide format. In general there are a host of preprocessing steps that people take before they can analyze their data. For instance, I used to have one file for every subject. When analyzing the data, I had to do computations on the individual file and store the outcome in a single overall file. Then I would run the analysis on that. It would be good to identify the most common preprocessing steps and provide some help with those -- but there is a limit to what we can do. It may also be relatively complicated but I'm not sure. Worth looking into.
